Controller/ContentManagement/README.md

```diff
@@ -0,0 +1,14 @@
+Using the rest API
```

Comment:
Don't like the readme inside the controller namespace. Resouces/doc folder should be usued for this

Reply:
I agree with @Davidmattei this file should be part of the Resources/doc filder and be named "api.md" and have a reference from the main readme file. There is allready an api.md file over there.

Reply:
Moved to the resources folder dd143f01f6bc2da86899bfaaea3c65185ec43a35